client/src/app/shared/shared-forms/select-options.component.ts

```diff
@@ -0,0 +1,47 @@
+import { Component, Input, forwardRef } from '@angular/core'
+import { NG_VALUE_ACCESSOR, ControlValueAccessor } from '@angular/forms'
+
+type Item = { id: number | string, label: string, description?: string }
```

Comment:
`SelectOptionsItem` and export it so parent components can directly use this type